setup.cfg

```diff
@@ -0,0 +1,13 @@
+
+# See the docstring in versioneer.py for instructions. Note that you must
+# re-run 'versioneer.py setup' after changing this section, and commit the
+# resulting files.
+
```

Comment:
This is missing here:

`[metadata]

description-file = README.md`

Reply:
will add